manifest/manifest.json

```diff
@@ -16,7 +16,7 @@
   ],
   "content_scripts": [
     {
-      "matches": ["file://*/*", "http://localhost/*", "https://*/*"],
```

Comment:
revert

Reply:
Removed 0.0.0.0 but kept 127.0.0.1 as it's been requested

Reply:
Fixed but I kept 127.0.0.1